File: Programmers/JAVA/lv1/lv1_가운데 글자 가져오기.java
https://programmers.co.kr/learn/courses/30/lessons/12903

class Solution {
    public String solution(String s) {
        String answer = "";
        
        int sLeng = s.length();
        if(sLeng%2 == 0)
            answer = s.substring(sLeng/2 - 1, sLeng/2 + 1);
        else
            answer = s.substring(sLeng/2, sLeng/2 + 1);
      
        return answer;
    }
}

File: Programmers/C#/lv1/lv1_가운데 글자 가져오기.cs
https://school.programmers.co.kr/learn/courses/30/lessons/12903?language=csharp

public class Solution
{
    public string solution(string s)
    {
        string answer = "";

        answer = (s.Length % 2 == 0) ? s.Substring(s.Length / 2 - 1, 2) : s[s.Length / 2].ToString();

        return answer;
    }
}